arcade/text.py

```diff
@@ -569,6 +569,50 @@ def position(self, point: Point):
             self._label.position = *point, self._label.z
 
 
+class CustomTextSprite(object):
+    def __init__(self, string, Alphabet_Textures, scale=1, 
+                 center_x=0, center_y = 0, 
+                 text_scale=1, text_margin=16, width=100, height = 40,  Background_offset_x=0, Background_offset_y=0, Background_scale=1, Background_Texture=None) -> None:
```

Comment:
Pythonic parameter/variable names never start with a capital letter. **Only constants are in full capitals, and only classes are in `CamelCase`**. Python mainly uses `snake_case`.

This line is also 172 characters, and that is too long. Try to keep this at eighty characters for readability.